python/py-pyobjc-cocoa/Portfile

```diff
@@ -2,61 +2,23 @@
 
 PortSystem          1.0
 PortGroup           python 1.0
+PortGroup           obsolete 1.0
 
 name                py-pyobjc-cocoa
-python.rootname     pyobjc-framework-Cocoa
 version             5.3
 revision            0
-categories-append   devel
-platforms           darwin
-license             MIT
-maintainers         nomaintainer
-
-description         cocoa wrappers for PyObjC
-long_description    This port provides the CoreFoundation, Foundation \
-                    and AppKit wrappers for use with PyObjC.
-homepage            https://pyobjc.readthedocs.io
-
-master_sites        pypi:p/${python.rootname}/
-distname            ${python.rootname}-${version}
-checksums           rmd160  5790e8357f6e7e5668b8750dd4809522ae44f812 \
-                    sha256  0b2d2f4a95d2634d1545c2d5cc949cb1b1bbfd73393d3c3b483be2a18906e568 \
-                    size    3808584
-
-python.versions     27 34 35 36 37 38
-
-if {${name} ne ${subport}} {
-    depends_build-append \
-                        port:py${python.version}-setuptools
-    depends_lib-append  port:py${python.version}-pyobjc
+epoch               1
 
-    patchfiles      pyobjc_setup.py.patch
+categories          python devel
+license             MIT
 
-    if {${configure.sdkroot} eq ""} {
-        set sdkroot "/"
-    } else {
-        set sdkroot ${configure.sdkroot}
-    }
+replaced_by         py-pyobjc
 
-    build.env-append    SDKROOT=${sdkroot}
-    destroot.env-append SDKROOT=${sdkroot}
+description         obsolete wrapper for PyObjC
 
-    platform darwin 8 {
-        depends_build
-        depends_lib
-        pre-fetch {
-            ui_error "${subport} @${version} requires Mac OS X 10.5 or later"
-            return -code error "incompatible Mac OS X version"
-        }
+foreach v {34 35 36 37 38} {
```

Comment:
please remove the py34/py35 subports; you cannot set it to be replaced_by py3[45]-pyobjc as that port doesn't define the py3[45] subports

Reply:
@reneeotten see above.